src/main/kotlin/sliv/tool/importer/view/ChoosingDirectoryView.kt

```diff
@@ -30,8 +31,12 @@ class ChoosingDirectoryView : Fragment() {
         }
```

Comment:
Add a button to send parsed project to the scene. Just call `sceneFacade.visualize(...)` from MainController and close the import window. What if import window was closed by Esc? Is is cancel?